python/GafferArnoldUI/ArnoldAttributesUI.py

```diff
@@ -249,9 +249,9 @@ def __toonSummary( plug ) :
 			"description",
 			"""
 			The lights that cause this object to cast shadows.
-			Accepts a set expression or a space separated list of
-			lights. Use \"defaultLights\" to refer to all lights that
-			contribute to illumination by default.
+
+			> Caution : This attribute has been superceded and will be removed. Use
+			> StandardAttributes to set the `shadowedLights` attribute instead.
```

Comment:
Another call for the Great Metadata Repatriation of 2025, but we should also update the description in `startup/GafferScene/arnoldAttributes.py` so the Attribute Editor description is consistent with this.